resources/js/features/games/components/GameAchievementSetsContainer/SetSelectionTabs/SetSelectionTabs.tsx

```diff
@@ -0,0 +1,84 @@
+import { type FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { route } from 'ziggy-js';
+
+import { BaseTooltip, BaseTooltipTrigger } from '@/common/components/+vendor/BaseTooltip';
+import { InertiaLink } from '@/common/components/InertiaLink';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { cn } from '@/common/utils/cn';
+
+import { GameAchievementSetTooltipContent } from '../../GameAchievementSetTooltipContent';
+import { useTabIndicator } from './useTabIndicator';
+
+interface SetSelectionTabsProps {
+  activeTab: number | null;
+}
+
+export const SetSelectionTabs: FC<SetSelectionTabsProps> = ({ activeTab }) => {
+  const { game, selectableGameAchievementSets } =
+    usePageProps<App.Platform.Data.GameShowPageProps>();
+  const { t } = useTranslation();
+
+  const initialActiveIndex = activeTab
+    ? selectableGameAchievementSets.findIndex((gas) => gas.achievementSet.id === activeTab)
+    : 0;
+
+  const { activeIndex, indicatorStyles, isAnimationReady, setActiveIndex, tabRefs } =
+    useTabIndicator(initialActiveIndex);
+
+  if (!selectableGameAchievementSets.length) {
+    return null;
+  }
+
+  return (
+    <div className="relative">
+      {/* Active Tab Indicator */}
+      <div
+        className={cn(
+          'absolute bottom-[-6px] left-0 h-[2px] bg-neutral-300 will-change-transform light:bg-neutral-800',
+          isAnimationReady ? 'transition-all duration-200 ease-out' : null,
+        )}
+        style={indicatorStyles}
+      />
+
+      {/* Tabs */}
+      <div className="relative flex items-center space-x-[6px]">
+        {selectableGameAchievementSets.map((gas, index) => (
+          <BaseTooltip key={gas.id}>
+            <BaseTooltipTrigger>
+              <InertiaLink
+                href={route('game2.show', { game: game.id, set: gas.achievementSet.id })}
+                prefetch="desktop-hover-only"
+                preserveScroll={true}
+                onClick={() => {
+                  setActiveIndex(index);
+                }}
+              >
+                <div
+                  ref={(el) => {
+                    tabRefs.current[index] = el;
+                  }}
+                  className={cn(
+                    'flex h-[30px] cursor-pointer items-center px-2 pb-[18px] pt-4 duration-300',
+                    'lg:active:translate-y-px',
+                    index === activeIndex
+                      ? 'text-white light:text-[#0e0e10]'
+                      : 'text-[#ffffff99] light:text-[#0e0f1199]',
+                  )}
+                >
+                  <img
+                    src={gas.achievementSet.imageAssetPathUrl}
+                    alt={gas.title ?? t('Base Set')}
+                    className="size-8 select-none rounded-sm"
+                  />
+                </div>
+              </InertiaLink>
+            </BaseTooltipTrigger>
```

Comment:
Accessing subset from http://localhost:64000/game2/3443 shows a blank page. Other games with empty subsets work, but do we want to show them?

```
select gas.game_id from game_achievement_sets gas join achievement_sets a
on a.id=gas.achievement_set_id where gas.type != 'core' and a.achievements_published = 0;
```